### DISCUSSION TITLE
Synapse support

### DISCUSSION THREAD

**Discussion by garyamorris** [hasura/graphql-engine#8681] [General] [answered]
Do you have any plans to support synapse? 

We're making a move to synapse and due to json support we're having to look at alternatives to Hasura and just wanted to see if there was any plans on extending the MS SQL Server support to include Synapse data warehouse.

**Comment by adas98012** [comment, marked as answer]
Thanks for your feedback. Hasura keeps adding support for new Databases based on market demand. Please keep following our product announcements .
